migration/transitioning.rst

```diff
@@ -0,0 +1,93 @@
+.. _transitioning_mt:
+
+Citus extends PostgreSQL with distributed functionality, but it is not a drop-in replacement that scales out all workloads. A performant Citus cluster involves thinking about the data model, tooling, and choice of SQL features used. Migrating an existing single-node PostgreSQL database to Citus sometimes requires adjusting the schema and queries for optimal performance.
+
+Migration tactics differ between the two main Citus use cases of multi-tenant applications and real-time analytics. The former is more clear cut so we'll begin there.
```

Comment:
Maybe remove the comment around the migration being more 'clear cut'? We could instead say something like, the multi-tenant model requires fewer data model changes.